### ISSUE TITLE
Display a list of active streamers currently in games on the home page

### ISSUE THREAD

**Issue by Cepheid-UK** [w3champions/w3champions-ui#279]
Coordinate with #211 

It would be nice to have a list of streamers sorted by viewers who are currently streaming and in w3c games on the right hand side of the home page, much like the Team Liquid website, e.g. below.

![unknown](https://user-images.githubusercontent.com/40397054/98937807-ddf39580-24de-11eb-91fc-f6810f2668ae.png)

**Comment by SergeyKhval** [comment]
Opened #305. Also [this PR](https://github.com/w3champions/w3champions-statistic-service/pull/103) needs to be merged before for this to work
